tiny_obj_loader.h

```diff
@@ -229,6 +229,7 @@ typedef struct {
 } mesh_t;
 
 typedef struct {
+  int smoothingGroupId;                          // Smoothing group id. 
```

Comment:
One main change for the fix. Adding an int parameter to shape_t structure. This is the only addition to the data structure.